rules/tournaments/aoe2de.md

```diff
@@ -0,0 +1,128 @@
+# Age of Empires 2
+
+## General Provisions
+
+These rules inherit all rules from the main rulebook.
+
+These rules apply to all participants of the "Age of Empires 2" tournament (hereinafter "AoE2" or "Game").
+
+The rules may be available in several languages. If the individual versions differ, the English version is considered the source of the truth.
+
+## Tournament Information
+
+### Tournament Platform
+
+The official tournament platform for this tournament is Battlefy.
+Please make sure that you have an account: https://battlefy.com/account/signup
+
+The official tournament page will be shared via the main communication channel established in the main rulebook.
+
+### Tournament Format
+
+The tournament is played in the following stages:
+
+| Stage   | Type               | Format    | Description                                                            |
+|---------|--------------------|-----------|------------------------------------------------------------------------|
+| Stage 1 | Swiss Rounds       | Bo1       | This stage will be seeded according to current 1v1 RM ELO rankings.    |
+| Stage 2 | Single Elimination | Bo1 / Bo3 | The bracket will be seeded according to the result of the first stage. |
+
+### Qualification Swiss System
+
+4 rounds are played according to the Swiss-System. The games will be reallocated after each round according to the
+current ranking. For the ranking the won and lost games count. The best eight players will then play for the tournament
+victory. If there is a tie for 8th place after the 4 rounds, the ranking will be calculated automatically.
+
+## Schedule
+
+The first appointment for the team representative will be the briefing on Saturday at 15:30.
+The tournament itself will start one hour later at 16:30.
+All players must be at the location at the start of the tournament.
+
+No accommodations will be made for other tournaments that may happen during the time of this tournament.
+
+## Settings
+
+### Game Version
+
+The game is played with the **latest version** of Age of Empires 2: Definitive Edition.
+
+### Server
+
+The game is played on the official "westeurope" server.
+
+### Lobby Settings
+
+The following settings must be set and checked before each game.
+
+![Lobby Settings for SwitzerLAN 23](https://cdn.discordapp.com/attachments/592823339643437056/1158747859139710987/lobby_settings.png "Lobby Settings")
+
+### Game Settings
+
+The following settings must be set and checked before each game.
+
+![Game Settings for SwitzerLAN 23](https://cdn.discordapp.com/attachments/592823339643437056/1158747858854482022/game_settings.png "Game Settings")
+
+### Mods
+
+All mods are allowed.
+
+## Game Account
+
+Each player plays with his own game account.
+
+## Map Pool
+
+* Arabia
+* Arena
+* African Clearing
+* Gold Rush
+* Four Lakes
+* Acropolis
+
+### Map Selection Process
+
+#### Swiss Rounds
+
+In the first round, the lower ranked (by 1v1 RM ELO) player has the choice of which map is played.
```

Comment:
Swiss rounds don't need seeding, so this should probably rewritten as

```suggestion
In the first round, the last named player can pick the map. 
```

Or alternatively, to make it fair, map banning or a roll of the dice 🎲